wafo/cycles/tp2rfc.m

```diff
@@ -82,8 +82,8 @@
 % Check input arguments
 ni = nargin;
 no = nargout;
-error(nargchk(1,4,ni));
-
+%error(nargchk(1,4,ni));

```

Comment:
All the commented %error(nargchk(??, ??; ni)) is no longer necessary and can be deleted.